Zend/zend_API.c

```diff
@@ -2774,12 +2774,15 @@ ZEND_API zend_class_entry *zend_register_internal_interface(zend_class_entry *or
 ZEND_API int zend_register_class_alias_ex(const char *name, size_t name_len, zend_class_entry *ce) /* {{{ */
 {
 	zend_string *lcname;
+	int persistent;
+
+	persistent = (!ce || ce->type != ZEND_USER_CLASS) ? 1 : 0;
```

Comment:
`ce` must be non-null here, so the check for `!ce` is not necessary. You can also save the ternary because the comparison already returns 0 or 1. So just `persistent = ce->type != ZEND_USER_CLASS` would be fine here.

Reply:
Good point! I'm updating the PR right now